### ISSUE TITLE
NTR: Muscle biopsy: phosphofructokinase level low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9074]
New term request Muscle biopsy: phosphofructokinase level low (segal_050203152502): PFK

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000359
- label: Reduced muscle 6-phosphofructokinase activity
- parents: [Abnormal enzyme activity in muscle tissue (HP:0034526)](https://hpo.jax.org/app/browse/term/HP:0034526)
- pmid: PMID:6227635
- def: Activity of the enzyme 6-phosphofructokinase in muscle tissue is below the lower limit of normal. 6-phosphofructokinase (EC 2.7.1.11) is also known as phosphofructokinase I and phosphohexokinase. The PFKM gene encodes the muscle isoform of phosphofructokinase. PFK catalyzes the irreversible conversion of fructose-6-phosphate to fructose-1,6-bisphosphate and is a key regulatory enzyme in glycolysis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9074
- synonyms:
